backend/config /db.js

```diff
@@ -1,18 +1,21 @@
-import mongoose from 'mongoose'
+import mongoose from "mongoose";
```

Comment:
I was getting an error in my termianl console from mongoose and this fixed it.
<img width="752" alt="Screen Shot 2022-07-12 at 10 15 58 PM" src="https://user-images.githubusercontent.com/28995548/179802701-1185f6c9-0ef9-40c1-89e5-af4ebb78633e.png">